pkgs/development/python-modules/protobuf/default.nix

```diff
@@ -19,8 +19,6 @@ buildPythonPackage {
   inherit disabled;
   doCheck = doCheck && !isPy27; # setuptools>=41.4 no longer collects correctly on python2
 
-  outputs = [ "out" "dev" ];
```

Comment:
dev is empty. 👍🏼

Reply:
I also read that currently python packages break when they have a dev output. So even if it weren't empty we probably should have dropped it. Cannot point you to the exact issue, sorry.